---
# Feel free to add content and custom Front Matter to this file.
# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults

layout: home
---

![image](/docs/cliPE_Logo_small-01.png)

Welcome to the homepage for curated loci prime editing (cliPE) method resources!

CliPE companion Shiny apps:

Click here to access the CliPEpy_1 shiny app for designing your cliPE libraries and epegRNA architectures for screening

Click here to access the CliPEr_1 shiny app for TEXT

Some of the shiny apps above require specific input files, see instructions here:

#
Clinvar missense tsv file

1. Navigate to https://www.ncbi.nlm.nih.gov/clinvar/ with Chrome or other web browser

2. Type gene name into search like 'TSC2'

3. On the left-side of the page, click the box next to 'Missense' in the Molecular consequence section

4. Just beneath 'Search results', click 'Download' to bring up the dropdown menu and click 'Create File' button

5. (optional) rename file to something like clinvar_mis_GeneName_DATE.tsv

6. upload tsv to shiny app CliPEpy_1 to design epegRNA libraries

#
gnomAD missense csv file

1. Navigate to https://gnomad.broadinstitute.org/ with Chrome or other web browser

2. Type gene name into search like 'TSC2'

3. Scroll down and just above 'configure table' button is a checkbox for 'Missense/inframe indel' Click the only button to the right

4. Click 'Export variants to CSV' button

5. (optional) rename file to something like gnomADmis_GeneName_DATE.csv

6. upload csv to shiny app CliPEpy_1 to design epegRNA libraries

If you have any questions or run into issues with any of the shiny apps, please contact jeffrey [dot] calhoun [at] northwestern [dot] edu.</p>
